scripts/data.js

```diff
@@ -247,12 +247,12 @@ let materials = [
 ];
 
 if (localStorage.getItem("mats") != null) {
-    tempList = JSON.parse(localStorage.mats);
-    for (x in tempList) {
-        if (!tempList[x] || !materials[x]) {
-            continue;
-        } else {
-            materials[x].qty = tempList[x].qty;
-        }
-    }
+    materials = JSON.parse(localStorage.mats);

```

Comment:
I see what's going on here.  

This is what I get when:
A: Not just writing this from scratch, and just modifying my ComponentCounter Code to work for this new thing.
B: Doing this after being awake for 36 hours, and was supposed to sleep 8 hours prior.